R/write.R

```diff
@@ -336,10 +335,10 @@ format_delim <- function(x, delim, na = "NA", append = FALSE,
 #' @export
 #' @rdname format_delim
 format_csv <- function(x, na = "NA", append = FALSE, col_names = !append,
-  quote = c("needed", "all", "none"),
-  escape = c("double", "backslash", "none"),
-  eol = "\n",
-  quote_escape = deprecated()) {
+                       quote = c("needed", "all", "none"),
+                       escape = c("double", "backslash", "none"),
+                       eol = "\n",
+                       quote_escape = deprecated()) {
```

Comment:
I think we are actually changing our style here to match what was already present vs. what styler has done FYI.

That being said, I'm OK with going along with styler for now. I just want to mention this, so you won't be surprised in the future.

Applies to several similar diffs.